stablehlo/tests/interpret_bitcast_convert.mlir

```diff
@@ -0,0 +1,224 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
+
+func.func @bitcast_convert_op_test_i1_to_i1() {
```

Comment:
IMO the implementation code  may not have to deal with each type, if we just copy the source bytes to destination. With that this test file will be simplified with few cases.

Reply:
nvm.